### ISSUE TITLE
train

### ISSUE THREAD

**Issue by hellosher** [jcdubron/scnn_pytorch#13]
Can you provide the format of the data used for training?Or could you please prefect the readme file?

**Comment by jcdubron** [comment]
Download CULane dataset and modify `train_data_dir` and `eval_data_dir` according to the root directory of CULane dataset.

**Comment by hellosher** [comment]
after i downloaded the dataset,there are some .txt files,including train.txt,train_gt.txt,val.txt,val_gt.txt,which files are needed in your code?sorry to bother you ,I don't quite understand your code

**Comment by jcdubron** [comment]
These files are lists of images and segmentation ground truth, which are used to generate dataloaders.

**Comment by hellosher** [comment]
does the train.txt need to change to include the ground truth label,just like this "/driver_23_30frame/05171102_0766.MP4/00020.jpg /laneseg_label_w16/driver_23_30frame/05171102_0766.MP4/00020.png 1 1 1 0",especially the "1 1 1 0"?Or we still need to include those raw files?
